tests/e2e/tests/AggregatesSection/AggregatesHasDynamicTitle.spec.ts

```diff
@@ -45,7 +45,7 @@ test(
     await dashboard.navigateToFloodDisasterType();
     // Assertions
     await aggregates.aggregateComponentIsVisible();
-    await map.turnOffLayer({ layerName: 'Glofas stations' });
+    await map.clickLayerCheckbox({ layerName: 'Glofas stations' });
```

Comment:
so the point why this isn't working is that the glofas stations layer does not toggle off when you click it? And subsequently when hovering over the map, it doesn't do that correctly? Why actually, as a user myself I could still hover over admin areas when the glofas station layer is on right? (let's indeed call to discuss further if needed)

Reply:
This is type of workaround that you have to perform while working with tools like Playwright, Cypress etc. Still when doing it manually I can see the title change but unfortunately when hovering with automation tool in the middle of the div or layer there are objects that interfere with this behaviour and that is why you'd like to turn off the layer (that is not tested in this case) to test the dynamic changes of aggregates header/title.

Reply:
Can you maybe add a note in the test explaining why it's skipped and what's needed to resolve it (maybe even link to specific GH issue?)

Reply:
Done